### DISCUSSION TITLE
Different color for executables

### DISCUSSION THREAD

**Discussion by Zikoel** [starship/starship#2251] [Q&A] [answered]
Hi,

I can't figure out how to use differently colour for files by its flags! For example I want to show executable files from `ls` with a different colour! Is something that is correct try to do with starship or simply this is not the purpose of the project? Should I search another way to do that ?

**Comment by mickimnet** [comment, marked as answer]
Hello Zikoel,

_Starship_ is only responsible for the display of the prompt, the part (above &) before the command/input line. If you're looking for colorful `ls` commands, I would recommend either [lsd](https://github.com/Peltoche/lsd) or [exa](https://the.exa.website).

Take care
Mick

**Reply by davidkna** [reply]
@Zikoel If you want to stick with plain `ls` you might want to have a look at [this](https://github.com/trapd00r/LS_COLORS). Depending on you system you might also want to alias `ls` to `ls --color` or `ls -G`.
